Packs/Doppel/Integrations/Doppel/Doppel.py

```diff
@@ -172,6 +194,48 @@ def update_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults
         outputs=result,
     )
 
+def get_alerts_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Command to fetch multiple alerts based on query parameters.
+
+    :param client: Client instance to interact with the API.
+    :param args: Command arguments containing the query parameters as key-value pairs.
+    :return: CommandResults object with the retrieved alerts.
+    """
+    # Extract query parameters
+    query_params = {
+        'search_key': args.get('search_key'),
+        'queue_state': args.get('queue_state'),
+        'product': args.get('product'),
+        'created_before': args.get('created_before'),
+        'created_after': args.get('created_after'),
+        'sort_type': args.get('sort_type'),
+        'sort_order': args.get('sort_order'),
+        'page': args.get('page'),
+        'tags': args.get('tags')
+    }
+
+    # Fetch results from the API
+    try:
+        results = client.get_alerts(params=query_params)
+        demisto.debug(f"Fetched alerts raw response: {results}")
+        if not results:
+            readable_output = "No alerts were found with the given parameters."
+        else:
+            readable_output = f"Retrieved {len(results)} alerts successfully.\n\nComplete JSON data:\n" \
+                              f"{json.dumps(results, indent=4)}"
+
+
+        return CommandResults(
+            outputs_prefix="Doppel.GetAlerts",
+            outputs_key_field="id",
+            outputs=results,
+            readable_output=readable_output,
+            raw_response=results
+        )
+    except Exception as e:
```

Comment:
Why this is a value error exception? It can be some other reason. Please check what standard practice for other commands.